docs/notes/2.24.x.md

```diff
@@ -24,7 +24,17 @@ We offer [formal sponsorship tiers for companies](https://www.pantsbuild.org/spo
 
 * Fixed bug where `pants peek --include-additional-info` was not actually displaying the additional info ([#21399](https://github.com/pantsbuild/pants/pull/21399)).
 
-### New Options System
+#### Internal Python Upgrade
+
+The version of Python used by Pants itself has been updated to [3.11](https://docs.python.org/3/whatsnew/3.11.html). To support this the [Pants Launcher Binary](https://www.pantsbuild.org/blog/2023/02/23/the-pants-launcher-binary-a-much-simpler-way-to-install-and-run-pants) known as  [`scie-pants`](https://github.com/pantsbuild/scie-pants/) now has a minimum version of `0.12.0`.  To update to the latest launcher binary run:
+
+```
+SCIE_BOOT=update pants
+```
+
+That Pants itself happens to be partially writtin in Python has no bearing on the versions of Python that Pants can use to test and build your code.
```

Comment:
Good to call this out.

Small typo:

```suggestion
That Pants itself happens to be partially written in Python has no bearing on the versions of Python that Pants can use to test and build your code.
```

But also, maybe we could be even more explicit with an example here, just to emphasise:

```suggestion
That Pants itself happens to be partially written in Python has no bearing on the versions of Python that Pants can use to test and build your code. For example, if you use Pants to build and test Python 3.9 code via `[python].interpreter_constraints == ["3.9.*"]`, this continues to work fine.
```

WDYT?